src/UI/examples/Input/Field/TextArea/with_limits.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This example shows how to create and render a basic textarea field with minimum and maximum number of characters limit.
+ * the input is attached to a form.
+ */
+function with_limits() {
+	//Step 0: Declare dependencies
+	global $DIC;
+	$ui = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+	$ctrl = $DIC->ctrl();
+	$request = $DIC->http()->request();
+
+	$min_limit = 3;
+	$max_limit = 20;
+
+	//Step 1: Define the textarea input field
+	$textarea_input = $ui->input()->field()->textArea("Textarea Input", "Just a textarea input.");
+	$textarea_input->withMinLimit($min_limit);
+	$textarea_input->withMaxLimit($max_limit);
+
+	//Step 2: Define the form and form actions.
+	$ctrl->setParameterByClass(
+		'ilsystemstyledocumentationgui',
+		'example_name',
```

Comment:
Please use another name than example_name here, due to the first example. So the display of the examples will run a little smoother.